etcdserver/api/membership/cluster.go

```diff
@@ -252,6 +252,24 @@ func (c *RaftCluster) Recover(onSet func(*zap.Logger, *semver.Version)) {
 	}
 }
 
+// IsPromoteChange checks whether cc is a promote change
+func (c *RaftCluster) IsPromoteChange(cc raftpb.ConfChange) bool {
+	members, _ := membersFromStore(c.lg, c.v2store)
```

Comment:
TODO (@jingyih): I think members are stored in c.be and they should be read from there. Need to take a closer look.